client/app/dedicated/server/translations/Messages_fr_FR.json

```
@@ -58,6 +58,7 @@
   "server_tab_stats_network": "Réseau :",
   "server_tab_stats_network_public": "Interface réseau publique ({{t0}})",
   "server_tab_stats_network_private": "vRack ({{t0}})",
+  "server_tab_stats_network_private_lag": "Interface privée aggrégée ({{t0}})",
```

```suggestion
  "server_tab_stats_network_private_lag": "Interface privée aggrégée ({{interface}})",
```
I think it's better to explicit what the expected translate argument is